packages/concerto-core/test/datetimeutil.js

```diff
@@ -32,7 +32,7 @@ describe('Initialize current time', () => {
         return currentTime.format().should.equal('1970-01-01T00:00:00+05:00');
     });
     it('Should fail for a non-well-formed date/time', function () {
-        return (() => DateTimeUtil.setCurrentTime('foobar')).should.throw('Cannot set current time to \'foobar\' with UTC offset \'undefined\'');
+        return (() => DateTimeUtil.setCurrentTime('foobar')).should.throw('Current time \'foobar\' is not in standard UTC format');
```

Comment:
Can you please add another test for the case where the offset is invalid?

Reply:
Done